web/src/js/components/common/MonacoEditor.jsx

```diff
@@ -0,0 +1,34 @@
+//not working
```

Comment:
hmm, it was a recommendation from max to use monaco. but `react-ace` looks great. if we use react-ace i can do this :)